apps/aecore/lib/aecore/peers/worker.ex

```diff
@@ -15,14 +15,14 @@ defmodule Aecore.Peers.Worker do
   require Logger
 
   def start_link(_args) do
-    GenServer.start_link(__MODULE__, %{}, name: __MODULE__)
+    GenServer.start_link(__MODULE__, %{peers: %{}, nonce: :rand.uniform(1000)}, name: __MODULE__)
   end
 
   ## Client side
 
   @spec add_peer(term) :: :ok | {:error, term()} | :error
   def add_peer(uri) do
-    GenServer.call(__MODULE__, {:add_peer, uri})
+    GenServer.call(__MODULE__, {:add_peer, uri}, 10000)
```

Comment:
why is this hardcoded?

Reply:
Because the request to self takes more than 5000ms, so we had to hardcode a higher value for the sake of the test we make

Reply:
ah okay, I did mistake it with the nonce